pkg/util/resourceapply/flowcontrol.go

```diff
@@ -109,6 +109,13 @@ func ApplyPriorityLevelConfiguration(ctx context.Context, getter flowcontrolclie
 		return current, false, nil
 	}
 
+	if current.Annotations != nil && current.Annotations["adapt-nominal-concurrency-shares"] == "true" {
```

Comment:
Usually annotations have a prefix that corresponds to the org defining it.

Is this annotation something we are adding as part of our operator?

Or is this an existing annotation?

Reply:
yes, I'm working on the e2e and this will also be documented.

Reply:
maybe we can follow our API naming for Kueue. 

something like `kueues.kueue.openshift.io/adapt-nominal-concurrency-shares`.

That way its clearly coming from our API and not an external annotation.